src/syscheckd/fim_db.c

```diff
@@ -596,6 +596,7 @@ int fim_db_process_read_file(fdb_t *fim_sql, fim_tmp_file *file, pthread_mutex_t
         i++;
     } while (i < file->elements);
 
+    fim_db_force_commit(fim_sql);
```

Comment:
I believe this statement is unnecessary since the callback functions are already making the commit. Besides, some callbacks don't require such operation, so it would be useless.

Reply:
Fixed f992c76